membrane_curvature/base.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
+
+
+class MembraneCurvature(AnalysisBase):
+
+    def __init__(self, universe, select='all',
+                 n_x_bins=100, n_y_bins=100,
+                 x_range=None,
+                 y_range=None,
+                 pbc=True, **kwargs):
+
```

Comment:
Probably already on your to-do list, but some docstring (especially detailing the input variables), would be very useful.

Reply:
Thanks, @IAlibay, I wasn't sure if it was going to be better/easier to open an issue after this PR was accepted. I will include docstrings in the next push.

Reply:
At least for this first PR I'd suggest adding input/output parameters and anything that's raised. That way it's easier for everyone to keep track of what is intended to be what.